migrations/000012_value_types.up.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN;
+
+SET search_path = public, pg_catalog;
+
+--
+-- Stores known metadata value types.
+--
+CREATE TABLE IF NOT EXISTS value_types (
```

Comment:
Should this table's initial data (now in `old/main/data/03_value_types.sql`) be added by this migration?
https://github.com/cyverse-de/metadata-db/blob/9907bd989e0d0328ac22f565a7981a8ae99e08b1/src/main/data/03_value_types.sql#L3-L14